console/console-init/ui/src/components/Editor/Editor.tsx

```diff
@@ -9,20 +9,34 @@ const Editor: React.FunctionComponent<IAceEditorProps> = ({
   name,
   onChange,
   enableBasicAutocompletion,
-  enableLiveAutocompletion
+  enableLiveAutocompletion,
+  width = "auto",
```

Comment:
why it is initialised at this point?

Reply:
width will take auto in case not coming as prop. On some screen we need Editor on full width and it was not taking full width.